windows/F8v2/MainPage.cs

```diff
@@ -73,7 +73,7 @@ public override bool UseDeveloperSupport
         private class F8Package : IReactPackage
         {
 #if !PRODFBAPP
-            private const string F8AppID = "100794426989995";
+            private const string F8AppID = "223301094807367";
```

Comment:
>223301094807367 [](start = 44, length = 15)

I think we should hold off on this change for now, it's probably not a good idea to check in these app IDs anyway...